virtcontainers/pkg/oci/utils.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/sirupsen/logrus"
 
 	vc "github.com/kata-containers/runtime/virtcontainers"
+	shim "github.com/kata-containers/runtime/virtcontainers/shim"
+	vtypes "github.com/kata-containers/runtime/virtcontainers/types"
```

Comment:
why jump around `shim` ,`vshim`, `types`,`vtypes` ？

Reply:
Thank you for your question.

Now it has been unified into vtypes and vshim.

vtypes "github.com/kata-containers/runtime/virtcontainers/types"

vshim "github.com/kata-containers/runtime/virtcontainers/shim"